parcel_test.go

```diff
@@ -117,15 +116,13 @@ func TestGetByClient(t *testing.T) {
 	}
 
 	var storedParcels []Parcel
-	storedParcels, err := store.GetByClient(client)
+	storedParcels, err = store.GetByClient(client)
 	require.NoError(t, err)
 	assert.Equal(t, len(parcels), len(storedParcels))
 
 	for _, parcel := range storedParcels {
+		_, ok := parcelMap[parcel.Number]
```

Comment:
Чисто визуально я бы тут сделал, чтобы два раза в мапу не лазить и `expectedParcel` сравнить с `parcel`

```suggestion
    expectedParcel, ok := parcelMap[parcel.Number]
```